source/frontend/src/components/maps/leaflet/mapUtils.tsx

```diff
@@ -242,17 +243,16 @@ export const asProperty = (point: PointFeature): IProperty => {
 
 /**
  * Convert any object to a cql filter string, assuming the object's keys should be used as CQL filter properties.
- * AND all object keys together within the generated cql filter string.
```

Comment:
This comment was wrong, if multiple search fields are provided (aka: pid, pin) we should OR them not AND them.